multicluster/reference/tap-values-iterate-sample.hbs.md

```diff
@@ -55,6 +59,7 @@ tap_telemetry:
 
 Where:
 
+Under the ca_cert_data key in the above terate-profile file, provide one or more PEM-encoded CA certificates if using custom CA certificates. If shared.ca_cert_data is configured, TAP component packages inherits that value by default.
```

Comment:
Fix the spell check. Iterate-profile file has become terate-profile.

Reply:
Thanks manohar. I will fix it.